addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1963,7 +1964,16 @@ export class Wysiwyg extends Component {
                 this.$editable.find(containerSelector)[0];
             if (container) {
                 const range = document.createRange();
-                range.selectNodeContents(container);
+                if (!container.hasChildNodes()) {
```

Comment:
You make the fix at the shortcut level, but what if the user right clicked, then clicked "select all"?

Reply:
I overlooked this scenario; i will investigate and address it properly.